wireless_modules/mqtt_test/main.py

```diff
@@ -0,0 +1,100 @@
+import time
+import machine
+from machine import Pin
+import ubinascii
+from config import MQTT_BROKER
+from umqttsimple import MQTTClient
+import fake
+
+# Define module number and MQTT topics
+MODULE_NUM = "1"
+FAKE_MODULE = fake.module(MODULE_NUM)
+
+PUB_DATA = b'/v3/wireless-module/{}/data'.format(MODULE_NUM)
+PUB_BATTERY = b'/v3/wireless-module/{}/battery'.format(MODULE_NUM)
+PUB_LOW_BATTERY = b'/v3/wireless-module/{}/low-battery'.format(MODULE_NUM)
+
+SUB_LED = b'/v3/wireless-module/{}/LED'.format(MODULE_NUM)
+SUB_TOPICS = [SUB_LED]
+
+# Define built in LED
+builtin_LED = Pin(2, Pin.OUT)
+
+# Set up MQTT client by generating a client_id
+client_id = ubinascii.hexlify(machine.unique_id())
+
+# MQTT callbacks
+def sub_cb(topic, msg):
```

Comment:
Not really fussed if this whole file isn't a class yet.

Reply:
Yeah this is also a bit hacky, but it works for now